### PR TITLE
Style changes for the home page

### PR DESCRIPTION
**Pull request by roberthouse54**
#### What are the relevant tickets?

#919 

#### What's this PR do?

This PR fixes the css styles for the Home Page

#### Where should the reviewer start?

See that the code looks good, and that the home page styling is not broken.

#### How should this be manually tested?

Maybe test in a couple browsers. NOTE: The layout is not responsive for mobile.

### CODE REVIEW

**Comment by roberthouse54** [comment]
Also NOTE: The final product ended up different than the mock. The reason for this was that Marketing requested a late color palette change to use MIT's maroon instead of the green we have had. To make these changes required other color palette changes. Now that the release got pushed back, I decided to stick with the green and use the extra time to rethink the color palette. But the preparatory changes I made in the design I am leaving in, because I think it looks good as is.

File: cms/templates/cms/home_page.html [hunk 1]
```diff
@@ -145,22 +155,23 @@
 
                   {% if program.programpage %}
                     {% if program.programpage.external_program_page_url %}
-                      <a href="{{ program.programpage.external_program_page_url }}" class="program-link">Learn More</a>
+                      <p><a href="{{ program.programpage.external_program_page_url }}" class="program-link">Learn More</a></p>p>
```
Comment (noisecapella) [inline cms/templates/cms/home_page.html]: Can you fix the `></p>p>` at the end here?

**Comment by noisecapella** [comment]
It looks like there are conflicts now. Do you want me to make the changes I mentioned and then merge this? The changes are pretty minor, the PR is generally in good shape

**Comment by Ferdi** [comment]
@noisecapella that would be great .

**Pull request merged**